logging/examples_test.go

```diff
@@ -86,6 +87,23 @@ func ExampleClient_Logger() {
 	_ = lg // TODO: use the Logger.
 }
 
+func ExampleHTTPRequest() {
+	ctx := context.Background()
+	client, err := logging.NewClient(ctx, "my-project")
+	if err != nil {
+		// TODO: Handle error.
+	}
+	lg := client.Logger("my-log")
+	httpEntry := logging.Entry{
+		Payload: "optional message",
+		HTTPRequest: &logging.HTTPRequest{
+			// TODO: pass in request
+			Request: &http.Request{},
+		},
+	}
+	lg.Log(httpEntry)
```

Comment:
Can you add a test for this?

Reply:
The example is the test and gets compiled. 
In this case, I dont expect outputs otherwise I will denote that with `// output: <output>`